recipe/meta.yaml

```diff
@@ -23,36 +27,61 @@ build:
 outputs: 
   - name: {{ name }}-examples
     build:
-      script: "%RECIPE_DIR%/build-examples.bat"
+      script: "%RECIPE_DIR%/build-examples.bat" #[win]
+      script: "$RECIPE_DIR/build-examples.sh" #[not win]
 
   - name: {{ name }}-documentation
     build:
-      script: "%RECIPE_DIR%/build-documentation.bat"
+      script: "%RECIPE_DIR%/build-documentation.bat" #[win]
+      script: "$RECIPE_DIR/build-documentation.sh" #[not win]
   
   - name: {{ name }}
     build:
-      script: "%RECIPE_DIR%/build-telemac.bat"
+      script: "%RECIPE_DIR%/build-telemac.bat" #[win]
```

Comment:
Indeed, I will try to rename it to bld.bat (should work)
Was cosmetic

Reply:
can be done after PR